{"uuid": "6568d99e-6369-4bb3-ab2b-5f2c8bdab6cb", "children": ["740af37b-187b-459a-bdd2-77eb866206c5", "9adf64e1-d9cd-4dcb-a9c6-82d3fc900ce9"], "befores": [{"name": "log_on_failure", "status": "passed", "start": 1737288420581, "stop": 1737288420581}], "afters": [{"name": "log_on_failure::0", "status": "broken", "statusDetails": {"message": "AttributeError: 'Class' object has no attribute 'rep_call'\n", "trace": "  File \"C:\\Users\\Vishal\\AppData\\Local\\Programs\\Python\\Python312\\Lib\\site-packages\\allure_commons\\_allure.py\", line 221, in __call__\n    return self._fixture_function(*args, **kwargs)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\Vishal\\AppData\\Local\\Programs\\Python\\Python312\\Lib\\site-packages\\_pytest\\fixtures.py\", line 922, in _teardown_yield_fixture\n    next(it)\n  File \"C:\\Users\\Vishal\\PycharmProjects\\pytestautomationdemo\\Pages\\LoginPage.py\", line 30, in log_on_failure\n    if item.rep_call.failed:\n       ^^^^^^^^^^^^^\n"}, "start": 1737288442904, "stop": 1737288442909}], "start": 1737288420581, "stop": 1737288442909}